about/glossary.rst

```diff
@@ -37,7 +56,7 @@ Glossary
       Digital Earth Australia, a platform that uses spatial data and images 
       recorded by satellites orbiting our planet to detect physical changes 
       across Australia in unprecedented detail. DEA is the Australian implementation of
-      the Open Data Cube. For more information see https://www.ga.gov.au/dea.
+      the Open Data Cube. For more information see https://www.dea.ga.gov.au/.
```

Comment:
try, ref for Open datacube https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#hyperlink-targets

Reply:
Here we want people to go to the DEA website. The ODC site is linked from there. The ODC link is linked in the ODC entry below.